e2e/tests/oai/002-oai-operators.yaml

```
@@ -64,3 +64,20 @@ spec:
     approval.nephio.org/policy: initial
   injectors:
   - name: edge
+---
+apiVersion: config.porch.kpt.dev/v1alpha2
+kind: PackageVariantSet
+metadata:
+  name: oai-ran-operator
+spec:
+  upstream:
+    repo: oai-ran-packages
+    package: oai-ran-operator
+    revision: v1
```

Are you sure that this is the right revision?
```suggestion
    revision: r2
```